test/users.spec.js

```diff
@@ -1,7 +1,27 @@
-import { expect } from 'chai';
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../server';
 
-describe('Sample test', () => {
-  it('should be true', () => {
-    expect(true).to.equal(true);
+chai.use(chaiHttp);
+
+describe('Authentication', () => {
+  describe('POST /api/users', () => {
+    it('should register a user', (done) => {
```

Comment:
Tests are not very useful if they only test the happy path. Your tests here are expected to cover every situation you've written code to handle.

Reply:
There's a separate story for that on the PT board – [#164139770](https://www.pivotaltracker.com/story/show/164139770)
I intend to write those tests there

Reply:
Good work on these by the way :)